lua/dap/utils.lua

```diff
@@ -34,25 +34,48 @@ end
 
 --- Show a prompt to select a process pid
 function M.pick_process()
-  local output = vim.fn.system({'ps', 'a'})
+  local is_windows = vim.fn.has('win32')
```

Comment:
```suggestion
  local is_windows = vim.fn.has('win32') == 1
```

`vim.fn.has(..)` returns 0 or 1 and both is truthy in lua